EpiJS/model.js

```diff
@@ -113,14 +113,15 @@ class Model {
 	remove(compartment) {
 		for (var x = 0; x < this.compartments.length; x++) {
 			if (this.compartments[x][0] === compartment) {
-				this.compartments.splice(x, 1)
+				this.compartments = this.compartments.splice(x, 1)
```

Comment:
This is an unnecessary change and it is inconsistent with the `add()` function which simply uses the `.splice()` method without assigning it to a variable.
```suggestion
				this.compartments.splice(x, 1)
```